internal/dinosaur/pkg/defaults/default_scanner_spec.go

```diff
@@ -1,57 +1,87 @@
+	Analyzer AnalyzerDefaults `envPrefix:"ANALYZER_"`
+	Db       DbDefaults       `envPrefix:"DB_"`
+}
+
 var (
-	// ScannerAnalyzerRequestMemory ...
-	ScannerAnalyzerRequestMemory = resource.MustParse("100Mi")
-	// ScannerAnalyzerRequestCPU ...
-	ScannerAnalyzerRequestCPU = resource.MustParse("250m")
-	// ScannerAnalyzerLimitMemory ...
-	ScannerAnalyzerLimitMemory = resource.MustParse("2500Mi")
-	// ScannerAnalyzerLimitCPU ...
-	ScannerAnalyzerLimitCPU = resource.MustParse("2000m")
+	// Scanner ...
+	Scanner ScannerDefaults
 	// ScannerAnalyzerResources ...
+	ScannerAnalyzerResources corev1.ResourceRequirements
+	// ScannerDbResources ...
+	ScannerDbResources corev1.ResourceRequirements
+)
+
+func init() {
+	defaults := ScannerDefaults{}
+	opts := env.Options{
+		Prefix: "SCANNER_",
+	}
+	if err := env.ParseWithFuncs(&defaults, CustomParsers, opts); err != nil {
+		panic(fmt.Sprintf("Unable to parse Central Defaults configuration from environment: %v", err))
+	}
+	Scanner = defaults
 	ScannerAnalyzerResources = corev1.ResourceRequirements{
 		Requests: map[corev1.ResourceName]resource.Quantity{
-			corev1.ResourceCPU:    ScannerAnalyzerRequestCPU,
-			corev1.ResourceMemory: ScannerAnalyzerRequestMemory,
+			corev1.ResourceCPU:    Scanner.Analyzer.CPURequest,
+			corev1.ResourceMemory: Scanner.Analyzer.MemoryRequest,
 		},
 		Limits: map[corev1.ResourceName]resource.Quantity{
-			corev1.ResourceCPU:    ScannerAnalyzerLimitCPU,
-			corev1.ResourceMemory: ScannerAnalyzerLimitMemory,
+			corev1.ResourceCPU:    Scanner.Analyzer.CPULimit,
+			corev1.ResourceMemory: Scanner.Analyzer.MemoryLimit,
 		},
 	}
-
-	// ScannerAnalyzerAutoScaling ...
-	ScannerAnalyzerAutoScaling = "Enabled"
-	// ScannerAnalyzerScalingReplicas ...
-	ScannerAnalyzerScalingReplicas int32 = 1
-	// ScannerAnalyzerScalingMinReplicas ...
-	ScannerAnalyzerScalingMinReplicas int32 = 1
-	// ScannerAnalyzerScalingMaxReplicas ...
-	ScannerAnalyzerScalingMaxReplicas int32 = 3
-
-	// ScannerDbRequestMemory ...
-	ScannerDbRequestMemory = resource.MustParse("100Mi")
-	// ScannerDbRequestCPU ...
-	ScannerDbRequestCPU = resource.MustParse("250m")
-	// ScannerDbLimitMemory ...
-	ScannerDbLimitMemory = resource.MustParse("2500Mi")
-	// ScannerDbLimitCPU ...
-	ScannerDbLimitCPU = resource.MustParse("2000m")
-	// ScannerDbResources ...
 	ScannerDbResources = corev1.ResourceRequirements{
 		Requests: map[corev1.ResourceName]resource.Quantity{
-			corev1.ResourceCPU:    ScannerDbRequestCPU,
-			corev1.ResourceMemory: ScannerDbRequestMemory,
+			corev1.ResourceCPU:    Scanner.Db.CPURequest,
+			corev1.ResourceMemory: Scanner.Db.MemoryRequest,
 		},
 		Limits: map[corev1.ResourceName]resource.Quantity{
-			corev1.ResourceCPU:    ScannerDbLimitCPU,
-			corev1.ResourceMemory: ScannerDbLimitMemory,
+			corev1.ResourceCPU:    Scanner.Db.CPULimit,
+			corev1.ResourceMemory: Scanner.Db.MemoryLimit,
 		},
 	}
-)
+}
+
+// PettyPrintScannerDefaults ...
+func PettyPrintScannerDefaults() []string {
```

Comment:
Did you mean PrettyPrint... instead of `Petty`?

Reply:
Apart from that, you could define a more generic function that takes an empty interface and a label and prints any object in that fashion instead of defining a function for scanner and central. The implementation itself would not differ much: (not tested)
```suggestion
func PettyPrintScannerDefaults(obj interface{}, label string) []string {
	bytes, err := json.MarshalIndent(obj, "", "  ")
	if err != nil {
		return nil
	}
	
	return append([]string{fmt.Sprintf("%s:", label)}, strings.Split(string(bytes), "\n")...)
}
```

Reply:
Thanks!